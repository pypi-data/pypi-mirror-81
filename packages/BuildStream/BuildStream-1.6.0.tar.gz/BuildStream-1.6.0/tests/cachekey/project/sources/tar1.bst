kind: import
sources:
- kind: tar
  url: https://example.com/releases/1.4/foo-1.4.5.tar.gz
  ref: 6c9f6f68a131ec6381da82f2bff978083ed7f4f7991d931bfa767b7965ebc94b
