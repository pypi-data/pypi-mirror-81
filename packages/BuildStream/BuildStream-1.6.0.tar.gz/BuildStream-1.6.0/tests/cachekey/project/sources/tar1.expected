f2360a3551f3cc5293cf9939773102bca6b6d0aa261bff78ef5f69aac3265afb